geppetto.js/geppetto-client/src/common/layout/LayoutManager.tsx

```diff
@@ -583,38 +501,11 @@ class LayoutManager {
     return this.tabsetIconFactory.factory(node.getConfig());
   }
 
-  /**
-   * Restore widget.
-   *
-   * @param widget
-   * @private
-   */
-  private restoreWidget(widget: Widget) {
-    const { model } = this;
-    widget.panelName = widget.defaultPanel;
-    const panelName = widget.panelName;
-    let tabset = model.getNodeById(panelName);
-    if (tabset === undefined) {
-      this.createTabSet(panelName, widget.defaultPosition, widget.defaultWeight);
-    }
-    this.moveWidget(widget);
-  }
 
-  private moveWidget(widget) {
-    const { model } = this;
-    model.doAction(
-      FlexLayout.Actions.moveNode(
-        widget.id,
-        widget.panelName,
-        FlexLayout.DockLocation.CENTER,
-        widget.pos
-      )
-    );
-  }
 }
 
-export function initLayoutManager(model, componentMap: ComponentMap, iconFactory: TabsetIconFactory) {
-  instance = new LayoutManager(model, componentMap, iconFactory);
+export function initLayoutManager(model, componentMap: ComponentMap, iconFactory: TabsetIconFactory, isMinimizeEnabled: boolean) {
```

Comment:
do you think it would be possible to enable/disable the panel with the widget configuration?